python/lsst/ts/FiberSpectrograph/csc.py

```diff
@@ -78,7 +78,7 @@ def report_summary_state(self):
             if self.summary_state in (salobj.State.ENABLED, salobj.State.DISABLED):
                 if self.device is None:
                     try:
-                        self.device = AvsFiberSpectrograph()
+                        self.device = AvsFiberSpectrograph(log=self.log)
```

Comment:
Do you want to make a child logger with its own sub-name? Or are you already doing that in the Spectrograph?

Reply:
Oh, good point. As written the spectrograph just logs directly to whatever is passed in to it. I think that's probably the best option, since it the spectrograph object doesn't know anything about where the logger is coming from, and the messages from it are part of what this CSC does.

Assuming this logger is called `FiberSpectrographCsc`, I think that's the correct log to send them to.

Reply:
That's fair. The Spectrograph does most of the work so it'll have most of the interesting log messages.